README.md

```diff
@@ -25,7 +25,7 @@ Core features include:
 <!-- toc -->
 
 - [Talawa Components](#talawa-components)
-- [Documentation](#documentation)
+- [Documentation -](#documentation--)
```

Comment:
Revert any change you make to this file

Reply:
Following changes were automatically added to the commit due the scripts running during commit 
and if i revert them once again while committing same change would be added.


https://github.com/PalisadoesFoundation/talawa-admin/assets/109027247/b0891956-311e-479d-a6f2-5422c893f3f0

had created the issue for same: #1624

Reply:
As the upstream (PalisadoesFoundation/develop) contains the above in README hence resolving